checkers/model_dunder_str.py

```diff
@@ -0,0 +1,51 @@
+import ast
+
+from .checker import Checker
+from .issue import Issue
+
+
+class DJ08(Issue):
+    code = 'DJ08'
+    description = '__str__ method should be present in all db models'
+
+
+class ModelDunderStrMissingChecker(Checker):
+
+    def checker_applies(self, node):
+        for base in node.bases:
+            if self.is_model_name(base) or self.is_models_attribute(base):
+                return True
+        return False
+
+    def is_model_name(self, base):
+        return (
+            isinstance(base, ast.Name) and
+            base.id == 'Model'
+        )
+
+    def is_models_attribute(self, base):
+        return (
+            isinstance(base, ast.Attribute) and
+            isinstance(base.value, ast.Name) and
+            base.value.id == 'models' and base.attr == 'Model'
+        )
```

Comment:
Given that this 3 methods are repeated from this [other module](https://github.com/rocioar/flake8-django/blob/master/checkers/model_form.py#L17), could you extract them both to another class, could be called `ModelChecker` that inherits from Checker, and make this 2 chekers inherit from it?

Reply:
Done. Can you check it?